src/azul/indexer/document.py

```diff
@@ -239,6 +240,7 @@ class NullableString(FieldType[Optional[str], str]):
     # Note that the replacement values for `None` used for each data type
     # ensure that `None` values are placed at the end of a sorted list.
     null_string = '~null'
+    unmapped_type = 'string'
```

Comment:
What I don't understand is why only NullableString is affected. As far as I understand the problem, it can happen for every field, so we need to provide unmapped_type for every field we sort by.

Reply:
good call, this also returned a 500 https://service.dev.singlecell.gi.ucsc.edu/index/samples?size=15&sort=organismAgeRange&order=asc&catalog=dcp2

Reply:
sorting by an organismAgeRange seems like it should not a valid sort option.

Reply:
Like we talked about today, there are various total orderings on intervals. Whether they make sense intuitively to the user is a different question, but it's a start